apicast/src/oauth/keycloak.lua

```diff
@@ -175,20 +175,20 @@ function _M.credentials(self, access_token)
   local jwt_obj, err = parse_and_verify_token(self, access_token)
 
   if not jwt_obj then
-    local err = "[jwt] failed to parse token: "..access_token
+    err = "[jwt] failed to parse token: "..access_token
     return nil, err
   else
     if jwt_obj.payload then
       local app_id = jwt_obj.payload.aud
-      
+
       ------
       -- oauth credentials for keycloak
       -- @field 1 Client id
       -- @field app_id Client id
       -- @table credentials_oauth
       return { app_id, app_id = app_id }, err
     else
-      local err = "[jwt] failed to parse token: "..jwt_obj.reason
+      err = "[jwt] failed to parse token: "..jwt_obj.reason
       return nil, err
```

Comment:
both of these assigns are useless and are just returned, so it can be just `return nil, "some message"`